### PR TITLE
Updates legacy url to updates.hypervisor.cloud.com

### PR DESCRIPTION
**Pull request by CitrixChris**

### CODE REVIEW

**Comment by danilo-delbusso** [comment]
shouldn't we just get rid of the legacy option? Is this a "just in case" approach?

**Review by danilo-delbusso — CHANGES_REQUESTED** [review]
We also need to bump the branding version in the `Jenkinsfile` to `v2.8`

File: Branding/Branding.resx [hunk 1]
```diff
@@ -118,7 +118,7 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="LEGACY_UPDATES_URL" xml:space="preserve">
-    <value>https://updates.xensource.com/XenServer/updates.xml</value>
+    <value>https://updates.hypervisor.cloud.com/xenserver/updates.xml</value>
```
Comment (kc284) [inline Branding/Branding.resx:121]: This now exposes the URL we didn't want to expose. Please revert it and add the value in the internal repo, and a line replacing https://updates.xensource.com/XenServer/updates.xml with the value in re-branding.sh.

**Review by kc284 — CHANGES_REQUESTED** [review]

File: Branding/Branding.resx (hunk 1 above)
Comment (kc284) [reply Branding/Branding.resx:121]: Please also add the ticket number in the commit message for tracking.

**Pull request closed without merging**